.github/workflows/publish-web.yaml

```diff
@@ -26,13 +26,13 @@ jobs:
 
       # Website setup
       - name: Setup hugo
-        uses: peaceiris/actions-hugo@v2.6.0
+        uses: peaceiris/actions-hugo@v2
```

Comment:
Ah yes. Dependabot. I think I changed to `v2` because I got bitten by another package that I did overspecify. I changed it back to v2.6.0 (which it used in 0.2.11)

Reply:
Thanks!